src/pages/project/new_project/EditProject.tsx

```diff
@@ -0,0 +1,16 @@
+import { useParams } from 'react-router-dom';
+import { StyledPageContent } from '../../../components/styled/Wrappers';
+import { IdentifierParams } from '../../../utils/urlPaths';
+import ProjectForm from './ProjectForm';
+
+const EditProject = () => {
+  const { identifier } = useParams<IdentifierParams>();
+
+  return (
+    <StyledPageContent>
+      <ProjectForm onClose={() => {}} identifier={decodeURIComponent(identifier)} />
```

Comment:
```suggestion
      <ProjectForm identifier={decodeURIComponent(identifier)} />
```

Ser for meg at når man trykker avbryt kan vi ta en `history.goBack()`. Det kan sikkert like godt gjøres i `ProjectForm` da, heller enn å sende som en prop.

Da vil man alltid komme tilbake til der man var, men spørsmålet blir hva vi skal gjøre om noen går direkte til oppretting av nytt prosjekt da, så mulig den trenger litt mer tenking 🤔

Vi har [oppg](https://sikt.atlassian.net/browse/NP-47279) på å gjøre samme på resultat-wizard, så for min del kan vi godt droppe "avbryt" her, og heller flytte det til den oppgaven

Reply:
Ja, jeg føler også at det er naturlig. Sparer litt tid på at vi ikke skal tenke ut hver vår strategi.